app/models/gpo_confirmation_code.rb

```diff
@@ -1,6 +1,8 @@
 class GpoConfirmationCode < ApplicationRecord
   self.table_name = 'usps_confirmation_codes'
 
+  self.ignored_columns = %(bounced_at)
```

Comment:
This is a single string, I think we want an array of strings
```suggestion
  self.ignored_columns = %w(bounced_at)
```

Reply:
why do i even use `%` literals? I have no idea how they work